src/middleware/aws/operation/dynamodb.rs

```diff
@@ -0,0 +1,149 @@
+use crate::{semcov, StringValue};
+
+use super::*;
+
+pub enum DynamoDBOperation {}
```

Comment:
I use [Empty Types](https://doc.rust-lang.org/nomicon/exotic-sizes.html#empty-types) to group sdk-specific methods, i.e.:

```rust
impl DynamoDBOperation {
    #[inline]
    pub fn describe_endpoints<'a>() -> AwsOperation<'a> {
        AwsOperation::dynamodb("DescribeEndpoints", vec![])
    }
}
```

See https://github.com/nentgroup/telemetry-rust/pull/15#discussion_r1520108318 for more details